### ISSUE TITLE
add functions to apply vaccination to next generation matrices

### ISSUE THREAD

**Issue by goldingn** [njtierney/conmat#40]
depends on #38

Once we have functions to generate next generation matrices, add functions to apply vaccination to them in the same way as in the `contact_matrices_australia` private repo. This could look something like this:
```
ngm_5y_fairfield_vacc_0.5 <- apply_vaccination(
	ngm = ngm_5y_fairfield,
	coverage_by_age = age_coverage,
	efficacy_susceptibility = efficacy_susceptibility,
	efficacy_infectiousness = efficacy_infectiousness
)
```

**Comment by njtierney** [comment]
Hey @goldingn Aarathy has implemented #38 in https://github.com/njtierney/conmat/pull/64

For applying vaccination, was this roughly the code to look at this?

https://github.com/goldingn/contact_matrices_australia/blob/main/lgas.R#L15

**Comment by njtierney** [comment]
I believe this has now been handled in #67 - just adding a NEWS bullet now
